src/parsec_client/operations/can_do_crypto.md

```diff
@@ -0,0 +1,54 @@
+# CanDoCrypto
```

Comment:
For the name, we can discuss. In the community meeting it was mentioned "CryptoCanDo" but then I thought if there is a new PSA Crypto operation named `psa_crypto_can_do` it is going to be PsaCryptoCanDo so might be weird.